code/modules/shuttle/supply.dm

```diff
@@ -538,16 +538,29 @@
 /datum/economy/simple_seller/tech_levels/begin_sell(obj/docking_port/mobile/supply/S)
 	temp_tech_levels = SSeconomy.tech_levels.Copy()
 
+/datum/economy/simple_seller/tech_levels/proc/getPrice(tech_rarity, tech_level, sold_level = null)
+	// Calculates tech disk's supply points sell cost
+	if(!sold_level)
+		sold_level = 1
+
```

Comment:
```suggestion
/// Calculates tech disk's supply points sell cost
/datum/economy/simple_seller/tech_levels/proc/getPrice(tech_rarity, tech_level, sold_level = 1)
```
two things here, first I moved your documentation to it's proper spot outside the proc. Second, I removed your if statement and instead put the default value to 1 which is what you were trying to do but with more logic. If the proc caller doesn't specify a value for `sold_level` the proc will automatically assign it a value of `1`

Reply:
that would... make sense

Reply:
ignore my comment lol